### ISSUE TITLE
How to get displayed header values

### ISSUE THREAD

**Issue by mikebski** [bootstrap-vue/bootstrap-vue#6461]
### Describe the bug

I cannot get an array of strings that represents the displayed, rendered column headers

I have a b-table and I'm writing an export function for the data to go to PDF.

I need a way to get the actual text that is rendered for each header, so far my export function does this:

```
exportTableData(row: any) {
    console.log("Export", row);
    console.log("Table", this.$refs.theTable)
    console.log("Table1.0", (this.$refs.theTable as any).fields)
    console.log("Table1.1", (this.$refs.theTable as any).computedFields)
    console.log("Table3", (this.$refs.theTable as any).computedItems)
  }
```
However, both fields and computedFields are objects:

```
0: {key: "name", sortable: true}
1: {key: "facilityCode", sortable: true, label: "Code"}
2: {key: "description", sortable: false}
3: {key: "fullContact", label: "Contact", sortable: true, filterByFormatted: true, formatter: ƒ, …}
4: {key: "status", sortable: true}
5: {key: "priority", sortable: true}
6: {key: "size", sortable: true}
```
How can I just get an array of strings to get only the text that is shown on the page (e.g. no sr-only text and with the correct caps/spacing (facilityCode -> Facility Code), etc...

In other words, I need to get an array of header values...

**Comment by Hiws** [comment]
Wouldn't it be simpler to just pass a `fields` array to `<b-table>`, where you define a `label` manually for each column and then use that array instead of accessing `<b-table>` through a ref?
